### PR TITLE
ps2avrgb: fix incorrect avr ports specified (for numlock and capslock LEDs)

### PR DESCRIPTION
**Pull request by krusli**
From #3382:

This one was my bad -- when I ported the backlighting code I didn't test the functionality with toggling the CapsLock key so I never noticed that I specified the wrong PINs for the backlighting.

NumLock should use D0 and CapsLock should use D1, but I incorrectly set them as D1 and D2 respectively in the code.

(original PS2AVRGB ports in a [header file](https://github.com/showjean/ps2avrU/blob/master/firmware/ps2avrGB/ledconfig.h) for reference)

#2634 already addressed the issue for the YMD96; I'll port them over back to the other PS2AVRGB boards and templates I updated with my backlighting code ([backlight.c](https://github.com/qmk/qmk_firmware/blob/master/keyboards/jj40/backlight.c), lines 20 and 21).

This should also fix the "JJ40 freezes when CapsLock is toggled" issue mentioned in #3199 and in this [Reddit thread](https://www.reddit.com/r/olkb/comments/8en8f1/strange_caps_lock_behavior/), although I'm not sure of the cause of the main issue there (only 3/5 underglow LEDs turning on).

@Oscillope can you test re-enabling the led_set_user code in your backlight.c? It should now work (previously toggling CapsLock would toggle the wrong pin (D2 instead of D1) as a PWM pin, causing the freeze).

Other updates:
- updated personal JJ40 keymap
- added personal DZ60 keymap

### CODE REVIEW

File: keyboards/dz60/keymaps/krusli/keymap.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#include "dz60.h"
```
Comment (drashna) [inline keyboards/dz60/keymaps/krusli/keymap.c]: Could this be replaced with `#include QMK_KEYBOARD_H`

File: keyboards/dz60/keymaps/krusli/config.h [hunk 2]
```diff
@@ -0,0 +1 @@
+#define USB_MAX_POWER_CONSUMPTION 100
```
Comment (drashna) [inline keyboards/dz60/keymaps/krusli/config.h]: You may want to add `#pragma once` at the beginning here.

File: keyboards/dz60/keymaps/krusli/keymap.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		_______, KC_VOLD, KC_VOLU, KC_MUTE, _______, _______, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, _______,
+		_______, KC_NO, 	_______, _______, _______, _______, _______, KC_PPLS, KC_PMNS, KC_END,  KC_PGDN, KC_DOWN, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______)
+};
+
+enum function_id {
+    SHIFT_ESC,
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (drashna) [inline keyboards/dz60/keymaps/krusli/keymap.c]: If this isn't being used, you could remove it (alse KC_GESC does this)

File: keyboards/dz60/keymaps/krusli/rules.mk [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
```
Comment (drashna) [inline keyboards/dz60/keymaps/krusli/rules.mk]: You only need this block. Everything above it is redundant and not needed.  

And the LAYOUT line at the bottom isn't needed, either.

**Comment by drashna** [comment]
Thanks!

**Comment by Oscillope** [comment]
@krusli Sorry for not replying sooner, I've been out of town for the weekend. I'll test re-enabling the caps lock led stuff tonight or tomorrow, will let you know! Thanks for looking into this!

**Pull request merged**